packages/client/components/MeetingSeriesRoot.tsx

```diff
@@ -0,0 +1,32 @@
+import React, {Suspense, useEffect} from 'react'
+import useQueryLoaderNow from '../hooks/useQueryLoaderNow'
+import useRouter from '../hooks/useRouter'
+import meetingSeriesRedirectorQuery, {
+  MeetingSeriesRedirectorQuery
+} from '../__generated__/MeetingSeriesRedirectorQuery.graphql'
+import MeetingSeriesRedirector from './MeetingSeriesRedirector'
+
+const MeetingRoot = () => {
+  const {history, match} = useRouter<{meetingSeriesId: string}>()
+  const {params} = match
+  const {meetingSeriesId: encodedMeetingSeriesId} = params
+  const meetingSeriesId = decodeURIComponent(encodedMeetingSeriesId)
+  useEffect(() => {
+    if (!meetingSeriesId) {
+      history.replace('/meetings')
+    }
+  }, [])
```

Comment:
+1 same here, better to use `<Redirect />`, although it'll have to be below usage of `useQueryLoaderNow` but that's ok